src/modules/twap/hooks/useTwapDiscreteOrders.ts

```diff
@@ -19,7 +19,8 @@ export function useTwapDiscreteOrders(): TwapToDiscreteOrders | null {
     return partOrders.reduce<TwapToDiscreteOrders>((acc, item) => {
       const order = orders?.[item.uid]
 
-      if (order) {
+      // Take only "real" orders
+      if (order && !order.composableCowInfo?.isVirtualPart) {
```

Comment:
The key to the problem was here, we took any part order, but we should take only "real"/"discrete" order